sample/src/main/java/com/canhub/cropper/sample/extend_activity/app/SExtendActivityJava.java

```diff
@@ -0,0 +1,34 @@
+package com.canhub.cropper.sample.extend_activity.app;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.util.Log;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.canhub.cropper.CropImage;
+
+public class SExtendActivityJava extends AppCompatActivity {
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        CropImage.activity()
+                .start(this);
+    }
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        if (requestCode == CropImage.CROP_IMAGE_ACTIVITY_REQUEST_CODE) {
+            CropImage.ActivityResult result = CropImage.getActivityResult(data);
+            if (resultCode == RESULT_OK) {
+                Uri photoUri = result.getUri();
+            } else if (resultCode == CropImage.CROP_IMAGE_ACTIVITY_RESULT_ERROR_CODE) {
+                Exception error = result.getError();
+                Log.e("err", error.getMessage());
+            }
+        }
+    }
+}
```

Comment:
Please take a look on `SCameraFragment` and copy it behaviour here.

This means we will have all 4 behaviours:
- startWithUri
- startWithoutUri
- startPickImageActivity       
- startActivityForResult

So we are sure all library methods are called in the Java.

Please use the MVP pattern, you can literally copy the kotlin pattern in your new java sample